From 4a9703bb8648eb6601d11e572d45462d269970e4 Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Tue, 10 Apr 2012 11:39:11 +0800
Subject: [PATCH 1298/2463] ENGR00177241-4 mx6 close APBH DMA clock when no
 I/O operation

Select APBH DMA automatically when enable GPMI NAND module.

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 drivers/mtd/nand/Kconfig |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/mtd/nand/Kconfig b/drivers/mtd/nand/Kconfig
index 2202ab6..e609b78 100644
--- a/drivers/mtd/nand/Kconfig
+++ b/drivers/mtd/nand/Kconfig
@@ -428,6 +428,8 @@ config MTD_NAND_GPMI_NAND
         depends on MTD_NAND && (SOC_IMX23 || SOC_IMX28 || SOC_IMX6Q)
 	select MTD_PARTITIONS
 	select MTD_CMDLINE_PARTS
+	select MTC_CHAR
+	select MXS_DMA
         help
 	 Enables NAND Flash support for IMX23 or IMX28.
 	 The GPMI controller is very powerful, with the help of BCH
-- 
1.7.7.4

